src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,108 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Cars {
+
+    private final List<Car> carList;
+
+    public static Cars from(String[] names, MovingValidator movingValidator) {
+        List<Car> cars = new ArrayList<>();
+        for (String name : names) {
+            Car car = new Car(new CarName(name), movingValidator);
+            cars.add(car);
+        }
+        return new Cars(cars);
+    }
+
+    public Cars(List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public int size() {
+        return this.carList.size();
+    }
+
+    public void move() {
+        for (Car car : this.carList) {
+            car.move();
+        }
+    }
+
+    public List<MovingDistance> movingDistances() {
+        List<MovingDistance> movingDistances = new ArrayList<>();
+        for (Car car : this.carList) {
+            movingDistances.add(car.movingDistance());
+        }
+
+        return movingDistances;
+    }
+
+    public Car get(int index) {
```

Comment:
```suggestion
    public Car find(int index) {
```

사소한 차이이고 별다른 차이는 없지만, 객체지향 사고로 전환하기에는 충분히 좋은 도전이라고 생각합니다.

Reply:
list를 사용 했기 때문에 해당 메소드명을 지었는데 만약 find같은 경우에는 java에서는 못본듯하고
jpa라던지 다른 라이브러리에서는 null을 반환 하는것을 보고 네이밍을 하였는데 이런 경우에는 null을 반환하는 내용을 추가하는게 맞을까요?